src/redux/store/configureStore.js

```diff
@@ -1,15 +1,16 @@
 import { combineReducers, createStore, applyMiddleware } from 'redux';
 import logger from 'redux-logger';
 import thunk from 'redux-thunk';
-
+import StocksReducer from '../reducers/stocksList';
+import ProfileReducer from '../reducers/ProfilesReducer';
+import SearchReducer from '../reducers/searchStock';
 
 const reducers = combineReducers({
-//   rockets: rocketsReducer,
-//   missions: missionsReducer,
+  profile: ProfileReducer,
+  searchedData: SearchReducer,
+  stocks: StocksReducer,
 });
 
-const store = createStore(
-  applyMiddleware(thunk, logger),
-);
+const store = createStore(reducers, applyMiddleware(thunk, logger));
```

Comment:
- `createStore` is deprecated. Please use `connect` instead

Reply:
Thank you @afanimartin  for the feedback, I could not find any function with that name associated with redux, however, I will use configureStore from the redux toolkit, it is not deprecated.